changes.go

```diff
@@ -703,15 +712,38 @@ func (s *ChangesService) getCommentInfoMapSliceResponse(u string) (*map[string][
 }
 
 // CreateChange creates a new change.
-// The change info ChangeInfo entity must be provided in the request body.
+//
+// The change input ChangeInput entity must be provided in the request body.
+//
 // Only the following attributes are honored: project, branch, subject, status and topic.
 // The first three attributes are mandatory.
+//
 // Valid values for status are: DRAFT and NEW.
 //
 // As response a ChangeInfo entity is returned that describes the resulting change.
 //
 // Gerrit API docs: https://gerrit-review.googlesource.com/Documentation/rest-api-changes.html#create-change
-func (s *ChangesService) CreateChange(input *ChangeInfo) (*ChangeInfo, *Response, error) {
+func (s *ChangesService) CreateChange(input *ChangeInput) (*ChangeInfo, *Response, error) {
+	u := "changes/"
+
+	req, err := s.client.NewRequest("POST", u, input)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	v := new(ChangeInfo)
+	resp, err := s.client.Do(req, v)
+	if err != nil {
+		return nil, resp, err
+	}
+
+	return v, resp, err
+}
+
+// CreateChangeDeprecated creates a new change. (DEPRECATED)
+//
+// Use CreateChange instead.
+func (s *ChangesService) CreateChangeDeprecated(input *ChangeInfo) (*ChangeInfo, *Response, error) {
```

Comment:
I think we can drop the full `CreateChangeDeprecated`.
Why? Because it will be a breaking change anyway.

Even if we introduce a `CreateChangeDeprecated`, people who update the library, need to change the call name.
If we remove `CreateChangeDeprecated` at all, they can directly change the input type from `ChangeInfo` => `ChangeInput`.

Due to the nature of (nearly) the same fields, this should be nearly no effort + they are on the right track anyway.
What do you think?

Reply:
Yes, that makes a lot of sense. Thanks for stripping it out. Sure, I'll go ahead and add those fields.